cdap-ui/app/cdap/components/Home/index.js

```diff
@@ -117,22 +130,19 @@ export default class Home extends Component {
           <Route
             exact
             path="/ns/:namespace/datasets/:datasetId/fields"
-            component={FieldLevelLineage}
-          />
-          <Route
-            exact
-            path="/ns/:namespace/datasets/:datasetId/fll-experiment"
             render={(props) => {
-              const FllExperiment = Loadable({
-                loader: () =>
-                  import(/* webpackChunkName: "FllExperiment" */ 'components/FieldLevelLineage/v2'),
-                loading: LoadingSVGCentered,
-              });
-              return (
+              const v2 = (
                 <ErrorBoundary>
                   <FllExperiment {...props} />
                 </ErrorBoundary>
```

Comment:
Should the error boundaries encapsulate both v1 and v2?

Reply:
Actually, it looks like the error boundary is not necessary around either since there is already an error boundary encapsulating Home. I've removed the extra error boundary.